plugins/com.google.cloud.tools.eclipse.ui.test/src/com/google/cloud/tools/eclipse/ui/AboutIniTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.eclipse.ui;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import com.google.cloud.tools.eclipse.test.util.EclipseProperties;
+import java.io.File;
+import org.junit.Rule;
+import org.junit.Test;
+
+/** Verify that */
```

Comment:
I'm fine with removing the Javadoc.

Reply:
Oops, and another bogon in the name of `testgetFeatureImage()`.